pyinjective/exceptions.py

```diff
@@ -28,3 +28,11 @@ class ConvertError(PyInjectiveError):
 
 class SchemaError(PyInjectiveError):
     pass
+
+
+class BannedAddressError(PyInjectiveError):
```

Comment:
I am not sure we really need to create new Exception classes. What is the intention for the two new exceptions?

Reply:
Please, see my comment above. I will replace these custom exception classes.